Sources/ArcGISToolkit/Components/Popups/PopupElementHeader.swift

```diff
@@ -37,6 +37,5 @@ struct PopupElementHeader: View {
                     .foregroundColor(.secondary)
             }
         }
-        .padding([.bottom], 4)
```

Comment:
On macOS the disclosure group's arrow is placed right next to the header with little spacing. Only workaround I can see would be to add leading padding here instead of nothing.

<img width="43" alt="Screenshot 2023-06-02 at 17 58 12" src="https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/22fe3e7b-58a0-4dd5-a639-65f873c88332">

Reply:
Yes, this is where I tried it successfully, but as Mark says, it'd be for Catalyst only.

Reply:
I added a little bit of padding, which does look better. I didn't want to add too much because then it looked, well, like too much!